vast/integration/reference/zeek-tsv-pipeline-format/step_10.ref

```diff
@@ -1,2 +1,15 @@
-error: !! syntax_error: zeek-tsv parser encountered invalid header line: separator '\x09' not found
- = note: from `parse "zeek-tsv"`
+#separator \x09
+#set_separator	,
+#empty_field	(empty)
+#unset_field	-
+#path	zeek.conn
+#fields	ts	uid	id.orig_h	id.orig_p	id.resp_h	id.resp_p	proto	service	duration	orig_bytes	resp_bytes	conn_state	local_orig	local_resp	missed_bytes	history	orig_pkts	orig_ip_bytes	resp_pkts	resp_ip_bytes	tunnel_parents	community_id	_write_ts
+#types	time	string	addr	port	addr	port	string	string	interval	count	count	string	bool	bool	count	string	count	count	count	count	vector[string]	string	time
+2009-11-18T08:00:21.486539	Pii6cUUq1v4	192.168.1.102	68	192.168.1.1	67	udp	-	163.82ms	301	300	SF	-	-	0	Dd	1	329	1	328	(empty)	-	-
+2009-11-18T08:08:00.237253	nkCxlvNN8pi	192.168.1.103	137	192.168.1.255	137	udp	dns	3.78s	350	0	S0	-	-	0	D	7	546	0	0	(empty)	-	-
+2009-11-18T08:08:13.816224	9VdICMMnxQ7	192.168.1.102	137	192.168.1.255	137	udp	dns	3.75s	350	0	S0	-	-	0	D	7	546	0	0	(empty)	-	-
+2009-11-18T08:07:15.800932	bEgBnkI31Vf	192.168.1.103	138	192.168.1.255	138	udp	-	46.73s	560	0	S0	-	-	0	D	3	644	0	0	(empty)	-	-
+warning: invalid Zeek header: #empty_field
+ = note: line 3
+warning: invalid Zeek header: #unset_field
```

Comment:
Is this a bug?

Reply:
These warnings are the output of the Zeek TSV parser.

I currently don't know exactly why the integration reference expects that output to be after the output of the Zeek TSV printer, but reading erroneous Zeek TSV files leads to the warning being printed first after any other output, so I decided to leave that for now.

Reply:
This reordering is a _feature_, not a bug, to prevent non-determinism in the integration test output. The behavior on the console or the explorer should still be what we want. 

We discovered some additional non-determinism that the current implementation does not take care of (when emitting diagnostics from multiple operators), but this should not be relevant here.

Reply:
I was more concerned about the actual warning *content*:

```
warning: invalid Zeek header: #unset_field
```

This *is* a valid Zeek header. (There multiple other occurrences of this same comment, just posting it once here though.)

Reply:
The warnings appear because the file that the Zeek TSV parser is parsing contains an `#unset_field` line without a value specified afterwards. Vice versa with `#empty_field`.

Is that you mean with "valid Zeek header"?

Reply:
Ah, got it! Then this comment is a no-op. Sorry.